ffi/java-idsdk/src/main/java/com/iotics/sdk/identity/experimental/ResolverClient.java

```diff
@@ -0,0 +1,109 @@
+package com.iotics.sdk.identity.experimental;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URL;
+import java.util.Base64;
+import java.util.Objects;
+
+public class ResolverClient {
+    private final URL base;
+    private final int DEFAULT_TIMEOUT = 2000;
+    private final OkHttpClient client;
+
+    public ResolverClient(URL base) {
+        this.base = base;
+        this.client = new OkHttpClient();
+    }
+
+    public final class Result {
+        private final String content;
+        private final String contentType;
+        private final boolean isErr;
+
+        public Result(String content, String contentType, boolean isErr) {
+            this.content = content;
+            this.contentType = contentType;
+            this.isErr = isErr;
+        }
+
+        @Override
+        public String toString() {
+            return "Result{" +
+                    "content='" + content + '\'' +
```

Comment:
Might this not make for a pretty large/ugly output? How about length of if + maybe some starting chars?

Reply:
the point is to "see" the result in its entirety.

if - in the future - it becomes too cumbersome one may do as you say and add a toFullString() for example. ATM it's fine